src/server/models/db/migrations/20230725094906_add_email_type_columns_to_tables/migration.sql

```diff
@@ -1,46 +1,66 @@
-BEGIN;
-
 -- CreateEnum
-CREATE TYPE "PostEmailType" AS ENUM (
-  'sendOneMonthPostEmail',
-  'sendOneWeekPostEmail',
-  'sendOneDayPostEmail',
-  'sendStartedPostEmail',
-  'sendUnpublishOneDayPostEmail',
-  'sendUnpublishWeeklyPostEmail'
+CREATE TYPE "AnnualReviewPostEmailType" AS ENUM (
+  'oneMonthBeforeStart',
+  'oneWeekBeforeStart',
+  'oneDayBeforeStart',
+  'started',
+  'oneDayBeforeUnpublish'
 );
```

Comment:
I've renamed this to "AnnualReviewPostEmailType" to prevent other emails from being added to the enum in future

Reply:
I also don't think it was clear whether or not emails should be before or after start date or before unpublish etc..